src/gui/torrentmodel.cpp

```diff
@@ -314,47 +298,53 @@ void TorrentModel::handleTorrentsUpdated()
 
 // Static functions
 
-QIcon getIconByState(BitTorrent::TorrentState state)
+static QString getIconId(BitTorrent::TorrentState state)
 {
     switch (state) {
     case BitTorrent::TorrentState::Downloading:
     case BitTorrent::TorrentState::ForcedDownloading:
     case BitTorrent::TorrentState::DownloadingMetadata:
-        return getDownloadingIcon();
+        return "downloading";
```

Comment:
`QString` constructor converts literals from UTF-8, while via `Qlatin1String` we specify that the literal is in Latin1 encoding, which is more efficient. When we drop Qt4, we will replace those with `QLatin1Literal`, which will be event more memory efficient.